tests/scientific/test_sonata_properties.py

```diff
@@ -14,8 +14,6 @@ def test_input_resistance():
     # create Node from config
     GlobalConfig.verbosity = LogLevel.VERBOSE
     n = Node(str(CONFIG_FILE))
-    # use edges file w/o plasticity
-    SimConfig.extra_circuits["All"]["nrnPath"] = EDGES_FILE + ":default"
```

Comment:
We don't need to restrict the population anymore?

Reply:
Tbh I dont know...
@WeinaJi ?

Reply:
Yes, I removed the extra edges file w/o plasticity  so it only tests with plasticity edges. Because the  w/o plasticity edges file is not included in the original `circuit_config.json`,  the `brainbuilder` extractor can not work on it. 
For this test, I don't see much difference on the test result.